TTC.Deployment.AmazonWebServices/Deployer.cs

```diff
@@ -35,8 +35,8 @@ public class Deployer
             
             if (!string.IsNullOrEmpty(_awsConfiguration.RoleName))

```

Comment:
Hmm yeah, this roleName configuration is needed but no test is covering it. I think I'll merge this PR and then fix the roleName bit before releasing a new version to nuget...